src/raftstore/store/peer.rs

```diff
@@ -139,6 +163,14 @@ pub struct ConsistencyState {
     pub hash: Vec<u8>,
 }
 
+enum RequestPolicy {
```

Comment:
It's nice to add some comments for each policy to describe briefly what/how is a local read, read index request, consistent read, consistent write, etc., since it helps to understand and these terms lives in related comments.